pkg/util/lifted/selectors/bimultimap.go

```diff
@@ -122,7 +122,7 @@ func (m *BiMultimap) Put(key Key, labels map[string]string) {
 			// No change to labels.
 			return
 		}
-		// Delete before readding.
+		// Delete before reading.
```

Comment:
This seems not read-ing, it's re-add-ing.

Reply:
got it, misjudgment.